server.js

```diff
@@ -1,7 +1,8 @@
-const express = require('express')
-const path = require('path')
-const app = express()
-const port = process.env.PORT || 5001
+const express = require('express');
```

Comment:
is server.js included in linting now? I didn't see a change in package.json.
Either way, should probably standardize on formatting so we don't have to review code style in prs :)

Even better if linting automatically fixed code when nodemon/watch picked up on file changes.

Reply:
> Any issues?

no issues at all :+1: